user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/GQIPage.md

```diff
@@ -17,3 +17,36 @@ Represents a single page of rows. Every query result consists of one or more suc
 | -------- | ---- | -------- | ----------- |
 | Rows | [GQIRow](xref:GQI_GQIRow)[] | Yes | Rows of the page. |
 | HasNextPage | boolean  | No | `True` if GQI can fetch additional pages, `False` if the current page is the last page. |
+
+> [!TIP]
```

Comment:
Feel like this tip has a lot of words but not really to the point.

Suggestion (to be chatGPTified):
Often not all data is required immediately to produce a query result. Therefore, consider splitting up rows in separate pages so they can be fetched and processed individually as needed to improve speed and memory efficiency. For example: [Link to example]